indexer-app/src/cmr/indexer/data/concepts/generic.clj

```diff
@@ -109,8 +110,9 @@
                       :long-name-lowercase (util/safe-lowercase long-name))
                doc)))))
 
-(defmethod esearch/parsed-concept->elastic-doc :generic
-  ;; Public function called by the indexer framework when a document is needed.
-  [context concept parsed-concept]
-  ;; context is not needed for this work, so call a local function without it
-  (parsed-concept->elastic-doc-without-context concept parsed-concept))
+(doseq [concept-type (concepts/get-generic-concept-types-array)]
+  (defmethod esearch/parsed-concept->elastic-doc concept-type
+    ;; Public function called by the indexer framework when a document is needed.
+    [context concept parsed-concept]
+    ;; context is not needed for this work, so call a local function without it
+    (parsed-concept->elastic-doc-without-context concept parsed-concept)))
```

Comment:
Is it this file that defines which fields to index or is it the indexer-app/resources/schemas/dataqualitysummary/v1.0.0/index.json?

Reply:
I will have to look if I can create the association fields for all generics, then I will try to put it there.